File: 1/1.go
package main

import "fmt"

const N = 1000

func main() {
	sum := 0
	for n := 1; n < N; n++ {
		if n % 3 == 0 || n % 5 == 0 {
			sum += n
		}
	}
	fmt.Println(sum);
}

File: 1/1.rs
const N: u64 = 1000;

fn main() {
    let mut sum: u64 = 0;

    for n in 3..N {
        if n % 3 == 0 || n % 5 == 0 {
            sum += n;
        }
    }

    println!("{}", sum);
}
